### ISSUE TITLE
db enters invalid state, causing many subsequent errors and requiring a reload

### ISSUE THREAD

**Issue by AntonOfTheWoods** [electric-sql/pglite#94]
I am using `pglite` in tandem with `electric`, while also adding a few tables and views.

In my install procedure I have a few occasions where a read query can be made before the objects (tables, views) are created. I am getting the following kinds of errors:

```
GoalsWidget.tsx:49 Uncaught (in promise) error: relation "non_fallback_only_view" does not exist
```

Which are then followed by:

```
electric-sql_pglite.js:4239 error performing snapshot: error: unrecognized node type: 67143218
...
chunk-OVRU7FYL.js:1 Uncaught (in promise) error: unrecognized node type: 67143218
```

Subsequent attempts to `CREATE` `VIEW`s then cause errors, though it seems indexes *can* get created... It also seems that most `INSERT` queries *do* execute, so my initialisation process can continue. When I get to a `CREATE OR REPLACE VIEW`, I then get a :

```
RuntimeError: null function or function signature mismatch
    at postgres.wasm
    at postgres.wasm
...
```

And 

```
electric-sql_pglite.js:4239 error performing snapshot: Infinity
Uncaught (in promise) Infinity
... many times then
electric-sql_pglite.js:4239 error performing snapshot: RuntimeError: Aborted(). Build with -sASSERTIONS for more info.
```

If I then reload everything and execute the `CREATE`s that failed, they all execute fine.

Basically, it appears that any sort of query error puts the db into an invalid state, and causes many things to break after that, with basically everything getting fixed with a complete page reload.

**Comment by AntonOfTheWoods** [comment]
I am also getting 

```
RuntimeError: memory access out of bounds
    at postgres.wasm:0x1bae3a
...
    at us (chunk-EKNQE2HU.js?v=f34be11b:4:14038)
    at postgres.wasm:0x491aae
```

After errors.

**Comment by samwillis** [comment]
Fixed in PGlite v0.2.0.alpha
